lib/matplotlib/backends/backend_svg.py

```diff
@@ -590,7 +595,7 @@ def _get_clip_attrs(self, gc):
         clippath, clippath_trans = gc.get_clip_path()
         if clippath is not None:
             clippath_trans = self._make_flip_transform(clippath_trans)
```

Comment:
Question: do we still need the `clippath_trans` value (this becomes input to the `self._make_id` method call at L607) if we have a distinct identifier for each clip-path?  I'm a little wary of collisions with the rectangle-bounds inputs, so it's nice to have _something_ that keeps the inputs partitioned, but `clippath_trans` may be expensive.  Even so: perhaps the correctness-first, performance-later principle is worth respecting here.

Reply:
Thanks - agreed.  I'd been considering relocating the `_make_flip_transform` call to the write phase, on the basis that it might de-duplicate a few calls.  However, I'm currently inclined to change as little of the code as possible, and potentially extract any cleanup into a subsequent PR.

Reply:
PRs are free and fewer lines of code makes review easier/faster :)